model-server/src/main/kotlin/org/modelix/model/server/CmdLineArgs.kt

```diff
@@ -21,6 +21,9 @@ internal class CmdLineArgs {
     @Parameter(names = ["-inmemory"], description = "Use in-memory storage", converter = BooleanConverter::class)
     var inmemory = false
 
+    @Parameter(names = ["--local-persistence"], description = "Use ignite with local disk persistence", converter = BooleanConverter::class)
+    var localPersistence = false
+
```

Comment:
Does the command line parsing have some feature like mutual exclusion to prevent users from providing `inmemory` and `--local-persistence` at the same time?

Why does this use POSIX double dashes instead of the single dash used for all other options?

Reply:
The other parameters should probably also use double dashes. I just dropped the commit as it is out of scope for this PR.

Reply:
Added it back, but also changed the names of the other parameters to use double dashes to fix the inconsistency.